packages/component/src/Styles/StyleSet/ThumbButton.ts

```diff
@@ -0,0 +1,93 @@
+export default function () {
+  return {
+    '&.webchat__thumb-button': {
+      appearance: 'none',
+      background: 'Transparent',
+      border: 0,
+      borderRadius: 2,
+      height: 16,
+      /* The Fluent icon is larger than the button. We need to clip it.
+      Without clipping, hover effect will appear on the edge of the button but not possible to click. */
+      overflow: 'hidden',
+      padding: 0,
+      width: 16,
+
+      '&:active': {
+        // background: 'var(--pva__palette__neutral-light)'
+        background: '#EDEBE9'
+      },
+
+      '&:focus': {
+        /* TODO: Verify with designer. This was #767676 (Gray ~120), this is now #8A8886 (Gray 110). */
+
+        // border: 'solid 1px var(--pva__palette__neutral-secondary-alt)'
+        outline: 'solid 1px #605E5C'
+      },
+
+      '& .webchat__thumb-button__image': {
+        /* TODO: Remove "color" if we want a different hover color. */
+        color: 'var(--webchat__color--accent)',
```

Comment:
nit: Having a tokens object similar to fluent would be nice.
```suggestion
        color: tokens.colorForegroundAccent,
```